src/quo2/components/record_audio/record_audio/view.cljs

```diff
@@ -0,0 +1,473 @@
+                                                    (let [locationX               (-> e .-nativeEvent.locationX)
+                                                          locationY               (-> e .-nativeEvent.locationY)
+                                                          pageX                   (-> e .-nativeEvent.pageX)
+                                                          pageY                   (-> e .-nativeEvent.pageY)
+                                                          moved-to-send-button?   (touch-inside-area?
+                                                                                   {:locationX locationX
+                                                                                    :locationY locationY}
+                                                                                   (send-button-area @ready-to-send?)
+                                                                                   true false false true)
+                                                          moved-to-delete-button? (touch-inside-area?
+                                                                                   {:locationX locationX
+                                                                                    :locationY locationY}
+                                                                                   (delete-button-area @ready-to-delete?)
+                                                                                   false true true false)
+                                                          moved-to-lock-button?   (touch-inside-area?
+                                                                                   {:locationX locationX
+                                                                                    :locationY locationY}
+                                                                                   (lock-button-area @ready-to-lock?)
+                                                                                   false false false false)
+                                                          moved-to-record-button? (and
+                                                                                   (touch-inside-area?
+                                                                                    {:locationX locationX
+                                                                                     :locationY locationY}
+                                                                                    record-button-area-big
+                                                                                    false false false false)
+                                                                                   (not= locationX pageX)
+                                                                                   (not= locationY pageY))]
+                                                      (cond
```

Comment:
In Clojure it's highly praised to break long functions into smaller ones. This often leads to more readable code and it's especially important for using the REPL. I'm not sure if you've tried and you didn't like, but you could consider extracting many of these anonymous functions, like `on-responder-move`, `on-responder-release` and so on.